prboom2/src/p_floor.c

```diff
@@ -638,6 +638,8 @@ int EV_DoFloor
         floor->floordestheight = P_FindLowestFloorSurrounding(sec);
         floor->texture = sec->floorpic;
 
+        if (!comp_floors)
```

Comment:
This is a bug. `comp_floors` is the entry in the comp enum and is a positive integer. So, `!comp_floors` always evaluates to `false`.

Reply:
Fixed in c47225a, thank you.